src/features/LoginModal/CreateAccountForm/CreateAccountForm.tsx

```diff
@@ -50,6 +52,27 @@ const CreateAccountForm: React.FC<CreateAccountFormProps> = (props) => {
         }    
     };
 
+    const loginWithGoogle = async () => {
+        setIsLoading(true)
+        try {
+            const user = await signInWithPopup(auth, new GoogleAuthProvider());
+            loginUser.mutate({ id: user.user.uid, email: user.user.email, displayName: user.user.displayName }, {
+                onError: (err: any) => {
+                    console.log(err)
+                },
+                onSuccess: () => {
+                    navigate('/dashboard/home')
+                    onClose()
+                }
+            })
+
+        } catch (err: any) {
+            setIsLoading(false)
+            setError(err.message);
+            console.log('error signing in', err.message);
+        }
+    };
+
```

Comment:
Should this function be moved to the LoginModal so that it is not having to be defined twice?